addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -336,7 +336,7 @@
             </field>
         </record>
 
-        <record id="action_approved_expense" model="ir.actions.act_window">
+        <record id="action_unsubmitted_expense" model="ir.actions.act_window">
```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/hr_expense/views/hr_expense_views.xml#L324